logspout/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gliderlabs/logspout:v3.2.3
+
+ENV ROUTE_URIS=logstash+tcp://log-agent:5610
+
+COPY ./start.sh /
+
+ENTRYPOINT ["/start.sh"]
```

Comment:
I might be wrong, but for **logspout** I'd roll with ```CMD``` instead of ```ENTRYPOINT```. 
Unless there are some options in the parent container or you have some plans to allow ```CMD``` to be used to customize the behaviour of the spout.

Reply:
You are right, changed